content/livesync/index.textile

```diff
@@ -1,6 +1,57 @@
 ---
 title: What is LiveSync?
+product: livesync
+languages:
+  - javascript
 ---
 
+p. The LiveSync product consists of three parts: the "Ably Realtime Data Models SDK":https://https://github.com/ably-labs/models,  "Ably Database Connector (ADBC)":https://github.com/ably-labs/adbc and an "Outbox Table":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html. LiveSync's purpose is to allow you to easily broadcast and receive realtime updates of an app's state through data stored on a database. LiveSync can support multiple users with concurrent updates and offers an optimistic UI for publishers, and a 'last write wins' strategy for transactional updates.
 
+This diagram illustrates how the "Ably Realtime Data Models SDK":https://https://github.com/ably-labs/models, "ADBC":https://github.com/ably-labs/adbc and "Outbox Table":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html work together:
 
+<a href="/images/diagrams/datasync-e2e-solution.png" target="_blank">
+  <img src="/images/diagrams/datasync-e2e-solution.png" style="width: 100%" alt="Basic authentication process diagram">
+</a>
+
+- User interaction := with the application's UI involves entering data stored in the local state.
+- Optimistic update := means the update immediately reflects the user's changes in the UI.
+- Local State := changes made by the user are first written to the application's local state, which then sends these changes to the server.
+- Server API := sends a write request to the server, which processes this request through an API endpoint. The server then attempts to write the changes in a transactional form to the database.
```

Comment:
"in a transactional form" sounds a bit strange, perhaps "writes the changes to the database in a transaction" ?